primer/index.bs

```diff
@@ -660,16 +616,53 @@ Response (content-type: application/json)
 
 <img src="primer-request.mmd.svg" width="980" />
 
-<h4 id="request-flow-step-1" class="no-num">1. An AJAX request is initiated</h4>
 
 In this example, Alice has logged into `https://decentphotos.example` and has completed the
 authentication steps above. She wants to make a request to Bob's Pod to get a photo album
 information at `https://bob.otherpod.example/private/photo_album.ttl`. Bob has previously
 granted access to Alice but has not granted access to anyone else.
 
+<h4 id="request-flow-step-1" class="no-num">1. Discover Authorization Server</h4>
+
+Client can discover Authorization Server by making request to the resource
+
+Request:
+```http
+GET https://bob.otherpod.example/private/photo_album.ttl
+```
+
+Response:
+```http
+HTTP/1.1 401 Unauthorized
+WWW-Authenticate: UMA realm="example",
+  as_uri="https://auth.otherpod.example",
+  ticket="016f84e8-f9b9-11e0-bd6f-0021cc6004de"
+```
+
+<h4 id="request-flow-step-1" class="no-num">1. Discover Token Endpoint</h4>
+
+Knowing Authorization Server now client can discover its Token Endpoint ([[!RFC8414]])
+
+Request:
+```http
+GET https://auth.otherpod.example/.well-known/uma2-configuration
+```
+
+Response:
```

Comment:
we should consider adding `uma_profiles_supported`
https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html#rfc.section.2

Reply:
tracked in #63